### PR TITLE
Configure Incoming Mail + user limit test

### PR DESCRIPTION
**Pull request by tarteo**
* Incoming email configuration
* Small test on user limit (just to increase test coverage :wink:)

### CODE REVIEW

File: container_accessibility/tests/test_user_limit.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
+from odoo.tools import config
+
+
+class TestUserLimit(TransactionCase):
+    def test_exceeded(self):
+        config["user_limit"] = "5"
+        with self.assertRaises(UserError):
+            for i in range(10):
```
Comment (MrGigSolutions) [inline container_accessibility/tests/test_user_limit.py:11]: Perhaps style, but it is not clear to me here why you iterate to 10. Why not first create 5 users, then test that creating the 6th raises an error? In theory it is possible that another setting causes this limit as well, so to be correct, you could also test that you are in fact allowed to create the user if you increase the limit.

File: l10n_nl_hr_expense/data/mail_alias_data.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="hr_expense.mail_alias_expense" model="mail.alias">
+        <field name="alias_name">uitgaven</field>
```
Comment (MrGigSolutions) [inline l10n_nl_hr_expense/data/mail_alias_data.xml:4]: Should this not be English with an entry in the nl po file?

File: scripts/setup_smtp.py [hunk 3]
```diff
@@ -20,18 +24,23 @@ def main(env, host, user, password):
...
-        "smtp_encryption": "starttls",
-        "smtp_port": 587,
+        "smtp_encryption": encryption,
+        "smtp_port": port,  # 587
         "sequence": 9999999,
         "smtp_host": host,
         "smtp_user": user,
         "smtp_pass": password,
         "private": True,
+        "from_filter": user.split("@")[1],
```
Comment (MrGigSolutions) [inline scripts/setup_smtp.py:34]: Do we need to allow for "stupid" users, who do not enter a proper email address?

**Review by MrGigSolutions — COMMENTED** [review]
Good work! I made a few small remarks, but if you are happy as it is, I am happy to approve it as is.

File: container_accessibility/tests/test_user_limit.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+                self.env["res.users"].create(
+                    {
+                        "name": "User %s" % i,
+                        "login": "User %s" % i,
+                        "groups_id": [
+                            self.ref("container_accessibility.group_restricted")
+                        ],
+                    }
+                )
+        config["user_limit"] = "0"
```
Comment (ByteMeAsap) [inline container_accessibility/tests/test_user_limit.py:21]: Would this affect the actual config file? I know its a strange question but would like to know more about it :) If yes, are we setting the user_limit to 0 by default?

File: container_accessibility/tests/test_user_limit.py (hunk 1 above)
Comment (tarteo) [reply container_accessibility/tests/test_user_limit.py:11]: I've created this test just to increase the coverage for now. By my priorities and limited time I can't create proper tests ATM.

I'v added a comment about this.

File: l10n_nl_hr_expense/data/mail_alias_data.xml (hunk 2 above)
Comment (tarteo) [reply l10n_nl_hr_expense/data/mail_alias_data.xml:4]: This is not a translatable field.

File: container_accessibility/tests/test_user_limit.py (hunk 4 above)
Comment (tarteo) [reply container_accessibility/tests/test_user_limit.py:21]: No just in memory the current worker, by default it's 0 so I just put it back to the default value here so other tests where users are created don't fail.

**Review by ByteMeAsap — APPROVED** [review]

File: scripts/setup_smtp.py (hunk 3 above)
Comment (tarteo) [reply scripts/setup_smtp.py:34]: In this case the user would be a devops person, I could add a regex to check if it's a correctly formatted address but I think it's a slim case as this is automatically generated (for Curq) anyways.

**Pull request merged**